lib/transactions/transaction.js

```diff
@@ -1,6 +1,21 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating transactions in Lisk Blockchain
```

Comment:
Transactions module provides functions for creating transactions for use with the Lisk Blockchain.